website/siteConfig.js

```diff
@@ -22,15 +22,123 @@ embed.register('stylesheet',
 
 /* List of projects/orgs using your project for the users page */
 const users = [
```

Comment:
The `pinned` property of users are not used.

Reply:
I've removed all `pinned` properties.